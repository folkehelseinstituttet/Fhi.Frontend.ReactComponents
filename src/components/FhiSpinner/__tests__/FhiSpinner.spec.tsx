import { render } from '@testing-library/react';
import Spinner from '../FhiSpinner';

describe('FhiSpinner', () => {
  test('snapshot test', () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
