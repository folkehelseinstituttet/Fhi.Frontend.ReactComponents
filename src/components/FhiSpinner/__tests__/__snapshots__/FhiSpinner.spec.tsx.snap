// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiSpinner snapshot test 1`] = `
<div
  className="spinner-border  "
  role="status"
>
  <span
    className="visually-hidden"
  >
    Loading...
  </span>
</div>
`;
